app/api/dao/mentorship_relation.py

```diff
@@ -124,20 +124,22 @@ def list_mentorship_relations(user_id=None, accepted=None, pending=None, complet
         Returns:
             message: A message corresponding to the completed action; success if all relationships of a given user are listed, failure if otherwise.
         """
-        if pending is not None:
-            return messages.NOT_IMPLEMENTED, 200
-        if completed is not None:
-            return messages.NOT_IMPLEMENTED, 200
-        if cancelled is not None:
-            return messages.NOT_IMPLEMENTED, 200
-        if accepted is not None:
-            return messages.NOT_IMPLEMENTED, 200
-        if rejected is not None:
-            return messages.NOT_IMPLEMENTED, 200
+        # To check if the entered 'state' is valid.
+        def isValidState(rel_state):
+            if rel_state == 'PENDING' or rel_state == 'ACCEPTED' or rel_state == 'REJECTED' or rel_state == 'CANCELLED' or rel_state == 'COMPLETED':
```

Comment:
what about having these values in a list and ask if `if rel_state in [value1, value2, ...]`? It would reduce this line's length

Reply:
Yes it's a good idea.